biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -1193,15 +1191,20 @@ subroutine LeafLayerPhotosynthesis(f_sun_lsl,         &  ! in
                  write (fates_log(),*)'gs_mol= ',gs_mol
                  call endrun(msg=errMsg(sourcefile, __LINE__))
               end if
-              
-              ! Compare with Ball-Berry model: gs_mol = m * an * hs/leaf_co2_ppress p + b
-              hs = (gb_mol*ceair + gs_mol* veg_esat ) / ((gb_mol+gs_mol)*veg_esat )
-              gs_mol_err = bb_slope(ft)*max(anet, 0._r8)*hs/leaf_co2_ppress*can_press + stomatal_intercept_btran
-              
-              if (abs(gs_mol-gs_mol_err) > 1.e-01_r8 .and.  (stomatal_model == 1)) then
-               write (fates_log(),*) 'CF: Ball-Berry error check - stomatal conductance error:'
-               write (fates_log(),*) gs_mol, gs_mol_err
-            end if
+             
+             ! Compare with Medlyn model: gs_mol = 1.6*(1+m/sqrt(vpd)) * an/leaf_co2_ppress*p + b
+              if ( stomatal_model == 2 ) then
+                 gs_mol_err = 1.6*(1 + medlyn_slope(ft)/sqrt(vpd))*max(anet,0._r8)/leaf_co2_ppress*can_press + stomatal_intercept_btran
```

Comment:
Thanks @Qianyuxuan , what is the 1.6, is that the diffusivity ratio of water to carbon?

h2o_co2_bl_diffuse_ratio

Reply:
wait, thats the boundary layer diffusion ratio, I think this is the stoma ratio:
h2o_co2_stoma_diffuse_ratio

Reply:
@rgknox Sorry. I forgot to change that. Yes. 1.6 is the factor that converts from conductance to CO2 to conductance to water vapour. I will adjust it soon.